Products/CMFPlone/profiles/default/metadata.xml

```diff
@@ -1,4 +1,4 @@
 <?xml version="1.0"?>
 <metadata>
-  <version>5019</version>
+  <version>5020</version>
```

Comment:
@datakurre the pull request jobs accept *multiple* pull requests per job (one per line), so if you paste both pull request URLs it will check out both branches. 🎉